addons/mail/static/src/models/messaging_menu/messaging_menu.js

```diff
@@ -19,7 +19,7 @@ registerModel({
          * mobile.
          */
         toggleMobileNewMessage() {
-            this.update({ isMobileNewMessageToggled: !this.isMobileNewMessageToggled });
+            this.update({ mobileNewMessageInputView: this.mobileNewMessageInputView ? clear() : insertAndReplace({ doFocus: true }) });
```

Comment:
this toggle does not work, because of the delete on blur, the click is always creating a new one with a flicker